vllm/engine/async_llm_engine.py

```diff
@@ -700,3 +696,22 @@ async def check_health(self) -> None:
         else:
             await self.engine.check_health_async()
         logger.debug(f"Health check took {time.perf_counter()-t}s")
+
+    def _shorten_for_logging(self, prompt,
+                             prompt_token_ids) -> Tuple[str, str]:
+        """Truncates the input prompt text and token ids for logging.
+        The string representations of both are truncated to a maximum length
+        of self.max_log_len, and then ellipses (...) are added to indicate
+        that the values were truncated.
+        """
+        if self.max_log_len is not None:
+            if prompt is not None and len(prompt) > self.max_log_len:
+                prompt = f"{prompt:.{self.max_log_len}}..."
```

Comment:
Is it worth logging the original prompt before truncation? (if we haven't done it elsewhere)

Reply:
Ah yeah only with debug (or something even lower if they have it?). But I guess they can always disable the truncation in that case 🤷

Reply:
I actually think debug logging is broken atm, looks like it might be fixed with: https://github.com/vllm-project/vllm/pull/2432